hw_1/vipcls data.R

```diff
@@ -109,5 +129,17 @@ quantile(vipcls$momage[vipcls$marstat=="widowed"], c(1), na.rm = TRUE)
 mean(vipcls$momage[vipcls$marstat=="never married"], na.rm = TRUE) 
 sd(vipcls$momage[vipcls$marstat=="never married"], na.rm = TRUE) 
 quantile(vipcls$momage[vipcls$marstat=="never married"], c(1), na.rm = TRUE)
-#patient IDs of the 5 subjects whose children had the smallest birthweights
-print(vipcls[order(vipcls$bw), ]   )
+
+# patient IDs of the 5 subjects whose children had the smallest birthweights
+# You need to actually find just the 5 participants whose children had the smallest
+# birthweights.Currently you're just sorting by bw.
+
+# Something like this:
+# Sort
+ordered_data <- vipcls[order(vipcls$bw), ]
+
+# Find the 5 patient ids on the top
+ptid_lbw <- ordered_data$patid[0:5]
+print(ptid_lbw)
```

Comment:
You had skipped this step before. Needed to sort the data and then find the patids for the 5 participants on the top.